dotnet/src/AutoGen.Anthropic/Middleware/AnthropicMessageConnector.cs

```diff
@@ -0,0 +1,132 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// AnthropicMessageConnector.cs
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.Threading.Tasks;
+using AutoGen.Anthropic.DTO;
+using AutoGen.Core;
+
+namespace AutoGen.Anthropic.Middleware;
+
+public class AnthropicMessageConnector : IStreamingMiddleware, IMiddleware
```

Comment:
`IStreamingMiddleware` is also an `IMiddleware`
```suggestion
public class AnthropicMessageConnector : IStreamingMiddleware
```

Reply:
Can you add a short-cut agent extension method for this connector because it's so widely used.

Check [here](https://github.com/microsoft/autogen/blob/4ebfb821860277d0960a1fd5b75bdb9b8e0ed8d5/dotnet/src/AutoGen.OpenAI/Extension/OpenAIAgentExtension.cs#L6) for an example